src/components/Common/Filters/QuickFilter.vue

```diff
@@ -64,7 +64,7 @@ export default {
       this.throttleSearch = true
       setTimeout(() => {
         this.throttleSearch = false
-      }, 200)
+      }, 50)
```

Comment:
boyscouting?

Reply:
Yes, they were inconsistency between the form and the url with a big timeout